site/src/components/DeploySettingsLayout/Sidebar.tsx

```diff
@@ -66,6 +66,9 @@ const SidebarNavItem: FC<
   );
 };
 
+// TODO: :axe:
+// do we really have like 5 of the same file?
```

Comment:
:D aggravating as it might be, can we move this to a new issue or internal cleanup list?